app/src/main/java/com/truongdc21/mediatree/di/DBModule.kt

```diff
@@ -0,0 +1,29 @@
+package com.truongdc21.mediatree.di
+
+import android.content.Context
+import androidx.room.Room
+import com.truongdc21.mediatree.data.database.MyDatabase
+import com.truongdc21.mediatree.data.database.SongDao
+import com.truongdc21.mediatree.utils.ConstantDatabase
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object DBModule {
+
+    @Singleton
+    @Provides
+    fun provideSongDatabase(
+        @ApplicationContext context: Context
+    ): MyDatabase =
+        Room.databaseBuilder(context, MyDatabase::class.java, ConstantDatabase.NAME_DATABASE).allowMainThreadQueries().build()
```

Comment:
```suggestion
        Room.databaseBuilder(context, MyDatabase::class.java, MyDatabase.NAME_DATABASE).allowMainThreadQueries().build()
```
Table name dùng 1 lần thì khai báo ra companion object của MyDatabase , cả version nữa nhé